config/.env.sample

```diff
@@ -15,17 +15,17 @@ LOG_LEVEL='INFO'
 
 # Specify the path to the parent folder of video you want to use, located inside the 'Captions' folder. 
 # The .srt files must be stored within this folder.
-VIDEO_TO_USE = ''
+VIDEO_TO_USE=''
```

Comment:
So what is the value `VIDEO_TO_USE` takes? based on issue #40 description it take a list.

Reply:
It is just the name of the folder within the Captions folder used. 
Issue #40 had an issue with the vector database not refreshing and testing itself to empty between two runs, so by assigning the name of the video used, it makes each video's vector database extracted from the transcript unique, so data from one transcript does not "leak" into another one. 

```
self.retriever = makeRetriever(
            self.videoData.combinedTranscript,
            self.LangChainQuestionBot.embeddings,
            self.config.videoToUse,
        )
```

We simply pass the video name when making the retriever for the vector database and it fixes the Issue.

Reply:
Ok, Got it.